### DISCUSSION TITLE
allow http_addr to bind to multiple addresses

### DISCUSSION THREAD

**Discussion by calestyo** [grafana/grafana#63523] [General]
Hey.

It seems one can specify only one address for `http_addr`. It would be nice if one could also specify multiple (e.g. separated by `,` or so) to exactly control to which addresses grafana should bind.

Thanks,
Chris.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82824. Feel free to continue the discussion around this there. Thank you!
